### PR TITLE
Write README File

### PR DESCRIPTION
**Pull request by EmanHass**

### CODE REVIEW

**Review by HebaAhmad1 — DISMISSED** [review]

**Review by MontherAlzamli — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+# Description:
+- It’s a list of tasks you need to complete, or things that you want to do.
+- One of the most important reasons you should use a todo list is that it will   help you stay organised. When you write all your tasks in a list, they seem more manageable. When you’ve got a clear outline of the tasks you’ve got to do and those you’ve completed, it helps you stay focused. While freeing up space in your mind for other more creative tasks.
+
+
+# Our Team:
+ * Eman Hassouna
+ * monther ismail
+ * Heba Al-Tanani
+
```
Comment (lina-jamal) [inline README.md:31]: add user stories, please.

Comment (lina-jamal) [reply README.md:31]: and add headline  : 
Live link [
How do I get the project on my local machine

**Review by lina-jamal — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (Sarah-Albalawi) [inline README.md:31]: Add my name to the team member, please

**Review by MontherAlzamli — DISMISSED** [review]

**Review by Sarah-Albalawi — APPROVED** [review]

**Review by Sarah-Albalawi — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
-# ToDoList-8
\ No newline at end of file
+# ToDoList Team 8
+
+# Description:
+- It’s a list of tasks you need to complete, or things that you want to do.
+- One of the most important reasons you should use a todo list is that it will   help you stay organised. When you write all your tasks in a list, they seem more manageable. When you’ve got a clear outline of the tasks you’ve got to do and those you’ve completed, it helps you stay focused. While freeing up space in your mind for other more creative tasks.
+
+# user stories:
+* Able to add his task.
```
Comment (lina-jamal) [inline README.md]: start user story with "As a user i can ..."

Comment (EmanHass) [reply README.md]: done

**Review by MontherAlzamli — DISMISSED** [review]

**Review by HebaAhmad1 — DISMISSED** [review]

**Pull request merged**